app/assets/javascripts/admin/addon/components/webhook-events.js

```diff
@@ -117,4 +142,43 @@ export default class WebhookEvents extends Component {
       popupAjaxError(error);
     }
   }
+
+  @action
+  async redeliverFailed() {
+    if (!this.failedEventIds.length) {
+      this.dialog.alert(
+        I18n.t("admin.web_hooks.events.no_events_to_redeliver")
+      );
+      this.redeliverEnabled = false;
+      return;
+    }
+
+    return this.dialog.yesNoConfirm({
+      message: I18n.t("admin.web_hooks.events.redeliver_failed_confirm", {
+        count: this.failedEventIds.length,
+      }),
+      didConfirm: async () => {
+        try {
+          const response = await ajax(
+            `/admin/api/web_hooks/${this.args.webhookId}/events/failed_redeliver`,
+            { type: "POST", data: { event_ids: this.failedEventIds } }
+          );
+          if (response.event_ids?.length) {
+            response.event_ids.map((id) => {
+              const event = this.events.findBy("id", id);
```

Comment:
This should be `this.events.find((event) => event.id === id);`, we are no longer using these ember array extensions. Sorry if I was the one who told you to do this in the first place! See /t/57006#avoid-embers-array-prototype-extension-methods-18 internally.